fdbserver/SimulatedCluster.actor.cpp

```diff
@@ -1390,7 +1409,11 @@ ACTOR void setupAndRun(std::string dataFolder, const char *testFile, bool reboot
 	state int minimumRegions = 0;
 	state int configureLocked = 0;
 	state int logAntiQuorum = -1;
-	checkTestConf(testFile, extraDB, minimumReplication, minimumRegions, configureLocked, logAntiQuorum);
+	state bool startIncompatibleProcess = false;
+	checkTestConf(testFile, extraDB, minimumReplication, minimumRegions, configureLocked, logAntiQuorum, startIncompatibleProcess);
+	g_simulator.hasDiffProtocolProcess = startIncompatibleProcess;
+	g_simulator.setDiffProtocol = false;
+	state ProtocolVersion protocolVersion = startIncompatibleProcess ? ProtocolVersion(currentProtocolVersion.version()+1) : currentProtocolVersion;
```

Comment:
I don't think this is different enough to actually make the protocols incompatible. I would add 0x10000 I think